MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/labs/LocationPickerActivity.java

```diff
@@ -262,11 +262,14 @@ public void onResponse(Call<GeocodingResponse> call, Response<GeocodingResponse>
             CarmenFeature feature = results.get(0);
 
             // If the geocoder returns a result, we take the first in the list and show a Toast with the place name.
-            if (style.isFullyLoaded() && style.getLayer(DROPPED_MARKER_LAYER_ID) != null) {
-              Toast.makeText(LocationPickerActivity.this,
-                String.format(getString(R.string.location_picker_place_name_result),
-                  feature.placeName()), Toast.LENGTH_SHORT).show();
-            }
+            mapboxMap.getStyle(new Style.OnStyleLoaded() {
+              @Override
+              public void onStyleLoaded(@NonNull Style style) {
+                Toast.makeText(LocationPickerActivity.this,
```

Comment:
Shall we check the `style.getLayer(DROPPED_MARKER_LAYER_ID) != null` here before showing the toast message?

Reply:
Sure. Null check added.